docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.9'
+services:
+  sb_db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password
+  sb_app:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3013 -b '0.0.0.0'"
+    volumes:
+      - .:/Switchboard
+    ports:
+      - "3013:3013"
+    depends_on:
+      - sb_db
+    extra_hosts:
+# This is what John did in his Docusign fix.
+#      - "app.myclio.ca:${DOCKER_HOST_IP:-127.0.0.1}"
+
+# Apparently in Docker 20.10 (which i'm using) you can do the below instead of resolving the host manually.
+      - "host.docker.internal:host-gateway"
+
+# Both John's and my version both work the same, you can enter a username/password and accept oauth scopes before
+# you get the following error:
+#   HTTP::ConnectionError in ApplicationController#identity_callback
+#   failed to connect: Connection refused - connect(2) for "account.myclio.ca" port 1337
```

Comment:
This means Switchboard is unable to resolve `account.myclio.ca`, similar to how Docusign was unable to resolve `app.myclio.ca`. The same fix could help here.